lib/services.js

```diff
@@ -478,6 +479,7 @@ export default {
         const mpuBucketName = `${constants.mpuBucketPrefix}${bucketName}`;
         metadata.getBucket(mpuBucketName, log, (err, mpuBucket) => {
             if (err) {
+                log.error('error from metadata', { error: err });
```

Comment:
For the method `metadata.getBucket` I will log an "error" if the `err` object is returned.